src/game/platform/keyboard.cpp

```diff
@@ -179,6 +179,8 @@ KeyASCIIType KeyboardClass::To_ASCII(uint16_t keycode)
     }
 
     return character[0];
+#else
+    return KeyASCIIType{};
```

Comment:
I've never seen this syntax for returning a default enum value before, should it not be KeyASCIIType(); to return the default value?

Reply:
No, because with `()` you can get yourself into [most vexing parse](https://en.wikipedia.org/wiki/Most_vexing_parse) situation. `{}` resolves any ambiguity and is part of C++11's uniform initialization.

Reply:
It's now been changed to `KA_NONE` anyway.

Reply:
C++ just keeps getting crazier and crazier :D